Resources/Prototypes/_EE/Entities/Mobs/Customization/Markings/human_hair.yml

```
@@ -89,3 +89,91 @@
+  markingCategory: Hair
+  sprites:
+    - sprite: _EE/Mobs/Customization/human_hair.rsi
+      state: stubbleshave
+
+- type: marking
+  id: HumanHairOvereyeponytail
+  bodyPart: Hair
+  markingCategory: Hair
+  sprites:
+    - sprite: _EE/Mobs/Customization/human_hair.rsi
+      state: overeyeponytail
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)